internal/pacseek/data.go

```diff
@@ -43,11 +43,21 @@ type InfoRecord struct {
 	DepsAndSatisfiers []DependencySatisfier
 }
 
+type PkgState int8
+
+const (
+	PkgNone      PkgState = 0x0
+	PkgInstalled          = 0x1
+	PkgMarked             = 0x2
+)
+
 type PkgStatus struct {
-	pkg       InfoRecord
-	installed bool
+	Pkg   InfoRecord
+	State PkgState
 }
 
+var pkglist []PkgStatus
```

Comment:
I don't know go that well, I need to access this variable in 2 different files. I think defining it on top of `setup.go` should work too, but not sure if proper place for it. Also I'm passing it as copy slice not as pointer, then functions return new modified slice ,not sure if it's deal-braker.